examples/hiera/common.yaml

```diff
@@ -0,0 +1,68 @@
+---
+## nsswitch configuration required for use with sssd
```

Comment:
Thanks @jeffmccune 

Could you please rename this from common.yaml to RedHat-6.yaml or RedHat-7.yaml, depending on to which platform this corresponds